readme/api/references/plugin_manifest.md

```diff
@@ -15,6 +15,7 @@ Name | Type | Required? | Description
 `repository_url` | string | No | Repository URL where the plugin source code is hosted.
 `categories` | string[] | No | [Categories](#categories) that describes the functionality of the plugin. 
 `screenshots` | Screenshot[] | No  | [Screenshots](#Screenshot) are used for listing on Joplin Plugin website.
+`icons` | Icons | No | If [Icons](#Icons) is not supplied, a standard extension icon will be used by default. You should supply at least a main icon, ideally 48x48 px in size. This is the icon that will be used in various plugin pages. You may, however, supply icons of any size and Joplin will attempt to find the best icon to display in different components. Icons should generally be in PNG format, because PNG has the best support for transparency. They can, however, be in any raster format supported by Blink, including BMP, GIF, ICO, and JPEG.
```

Comment:
I've used the same text from [Chrome Extensions Manifest - icons](https://developer.chrome.com/docs/extensions/mv3/manifest/icons/) to describe the extensions of the icons.

In case you want more info about raster formats and Blink, please, follow:
[Raster Format](https://www.adobe.com/creativecloud/file-types/image/raster.html)
[Blink](https://www.chromium.org/blink/)

Reply:
Ok but try to think about how the icons are going to be used. They will be displayed in the plugin website for example, and we want to support more than just Chrome, right? So Blink has no relevance here. In fact unnusal formats like ICO or BMP might be a problem depending on where the icons are displayed - for example if we eventually have plugins in mobile, I'm not sure that React Native support this by default. So for now let's restrict it to PNG.

Reply:
Sure. Let's simplify. Thanks for the explanation. In your opinion do we need to validate the extension of the icon when they're provide or will we do it when using it?